mopidy/core/library.py

```diff
@@ -227,9 +227,11 @@ def lookup(self, uri=None, uris=None):
         results = {u: [] for u in uris}
 
         # TODO: lookup(uris) to backend APIs
-        for backend, backend_uris in self._get_backends_to_uris(uris).items():
-            for u in backend_uris:
-                futures[(backend, u)] = backend.library.lookup(u)
+        futures = {
```

Comment:
I actually totally agree, I pretty much copied it from the other similar place it was used.